imagej/__init__.py

```diff
@@ -620,15 +620,13 @@ def to_java(self, data):
             :param data: Python object to be converted into its respective Java counterpart.
             :return: A Java object convrted from Python.
             """
-            if self._is_memoryarraylike(data):
-                return imglyb.to_imglib(data)
-            if self._is_xarraylike(data):
```

Comment:
We need to keep the separate xarray case. That is: xarray should convert to `Dataset`, while numpy array should convert to `Img`.

Reply:
Yup, I noticed that before you even wrote this 😅

Reply:
Fixed in 4afce29

P.S. the tests broke on the previous commit, so we should probably squash those.